### DISCUSSION TITLE
How are the beautiful svg animations made?

### DISCUSSION THREAD

**Discussion by KorigamiK** [zed-industries/zed#13012] [Feature Requests]
Animations that can be seen throughout the docs and the website are gorgeous and I wanted to know how they were made so clean?

Example: https://zed.dev/blog/videogame

**Comment by mlevkov** [comment]
@KorigamiK 
I think you can find details on this topic in the following discussion -> https://github.com/zed-industries/zed/discussions/6555

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
